client/app/telecom/pack/xdsl/tasks/pack-xdsl-tasks.html

```diff
@@ -5,13 +5,21 @@
         <toast-message></toast-message>
 
         <div class="widget-presentation">
-            <h2 class="widget-presentation-title"
-                data-translate="xdsl_tasks_list">
+            <h2 class="widget-presentation-title">
+                <span data-translate="xdsl_tasks_list"></span>
+                <button type="button" class="btn btn-default" data-ng-click="$ctrl.refresh()" 
```

Comment:
```suggestion
                <button type="button" class="oui-button oui-button_secondary oui-button_icon-only" data-ng-click="$ctrl.refresh()" 
```

Reply:
It doesn't render the same style that I need.

Reply:
The `oui-button` is the style that we definitely need. I'm agree with what @marie-j said.

Moreover I think that we could apply the same UI experience that was done a that case for instance (https://github.com/ovh-ux/ovh-manager-cloud/blob/master/client/app/kubernetes/nodes/kubernetes-nodes.html#L37-L48).

We have to provide a unify way of doing things across the entire application.